es/criteria/continuous-integration.md

```diff
@@ -32,33 +32,33 @@ order: 12
 * Probar situaciones de la vida real.
 * Garantizar que el código ofrezca exactamente el mismo resultado de las políticas.
 
-## Cómo testear
+## Cómo probar
 
-* Hay tests presentes.
+* Hay pruebas presentes.
 * Las herramientas de cobertura del código comprueban si la cobertura es del 100% del código.
-* Las contribuciones sólo se admiten en la codebase después de que se hayan superado todos los tests.
+* Las contribuciones son aceptadas en la codebase sólo después de que se hayan superado todas las pruebas.
```

Comment:
Like the reformulation of the sentence. Concerning the last bit of it perhaps we can use "hacer pruebas o tests"